cfme/tests/cloud_infra_common/test_html5_vm_console.py

```diff
@@ -231,24 +232,6 @@ def test_html5_console_ports():
     pass
 
 
-@pytest.mark.manual
```

Comment:
Note: Removed a number of manual test cases as I am refactoring what we test and we don't.